src/smt/theory_str_noodler/theory_str_noodler.cpp

```diff
@@ -2743,7 +2746,7 @@ namespace smt::noodler {
         }
 
         for(const auto& nc : this->m_not_contains_todo_rel) {
-            app_ref nc_app(m_util_s.str.mk_contains(nc.first, nc.second), m);
+            app_ref nc_app(m.mk_not(m_util_s.str.mk_contains(nc.first, nc.second)), m);
```

Comment:
This is just a fix, there should have always been `mk_not`, right?

Reply:
Yes